client/src/app/users/add-user.component.spec.ts

```diff
@@ -282,7 +281,7 @@ describe('AddUserComponent#submitForm()', () => {
         MatSelectModule,
         MatInputModule,
         BrowserAnimationsModule,
-        RouterTestingModule.withRoutes([
+        RouterModule.forRoot([
```

Comment:
Oh, nice! This turned out to be quite easy to deal with, where I had worried that it was going to be a big complicated thing. Well done!